src/Bindings/PluginManager.cpp

```diff
@@ -166,34 +166,51 @@ void cPluginManager::InsertDefaultPlugins(cSettingsRepositoryInterface & a_Setti
 void cPluginManager::Tick(float a_Dt)
 {
 	// Unload plugins that have been scheduled for unloading:
-	AStringVector PluginsToUnload;
+	decltype(m_PluginsNeedAction) PluginsNeedAction;
 	{
-		cCSLock Lock(m_CSPluginsToUnload);
-		std::swap(m_PluginsToUnload, PluginsToUnload);
+		cCSLock Lock(m_CSPluginsNeedAction);
+		std::swap(m_PluginsNeedAction, PluginsNeedAction);
 	}
-	for (auto & folder: PluginsToUnload)
+
+
+	for (auto & currentPlugin: PluginsNeedAction)
```

Comment:
```suggestion
	for (auto & CurrentPlugin: PluginsNeedAction)
```
We use CamelCase for local variable names.